lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -43,24 +42,21 @@ class PostgreSqlPlatform extends AbstractPlatform
     /** @var bool */
     private $useBooleanTrueFalseStrings = true;
 
-    /** @var string[][] PostgreSQL booleans literals */
+    /** @var bool[] PostgreSQL booleans literals */
     private $booleanLiterals = [
```

Comment:
Speaking of the optimization, would it make sense to make it a constant instead of an object property?